schema/schema_test.go

```diff
@@ -40,6 +40,33 @@ func ExampleSchema_Decode() {
 	// {Name:Bar Age:43}
 }
 
+func ExampleSchema_Decode_withTags() {
```

Comment:
I would suggest for that being the actual ExampleSchema_Decode (removing the method above) and add a small comment at line 55 saying that removing the struct tag will make the field name match the column name. What do you think?

Same for encoding, decode/encoding table.

What do you think?

Also, could you please add unit tests for struct tags to TestDecode/TestEncode bellow? We never know when those examples are going to change :D

Reply:
Yes. Good point.